src/amo/components/AddonsCard/styles.scss

```diff
@@ -0,0 +1,89 @@
+@import "~amo/css/inc/vars";
+@import "~core/css/inc/mixins";
+@import "~ui/css/vars";
+
+.AddonsCard--horizontal {
+  @include respond-to(large) {
+    position: relative;
+
+    .Card-contents {
+      background: $white;
+      border-bottom-left-radius: $border-radius-default;
+      border-bottom-right-radius: $border-radius-default;
+      padding: 0 20px;
+    }
+
+    .Card-footer,
+    .Card-footer-link {
+      @include end(20px);
+      @include text-align-end();
+
+      background: none;
+      border: 0;
+      position: absolute;
+      top: 0;
+    }
+
+    ul.AddonsCard-list {
```

Comment:
Sadly yes because the `CardList` styles are a bit too specific. We can clean them up later but for now this is fine 😄